spring-boot/ServletReg/src/main/java/com/baeldung/configuration/WebMvcConfigure.java

```diff
@@ -0,0 +1,38 @@
+package com.baeldung.configuration;
+
+import org.springframework.boot.web.support.ErrorPageFilter;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.ViewResolver;
+import org.springframework.web.servlet.config.annotation.DefaultServletHandlerConfigurer;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+import org.springframework.web.servlet.resource.PathResourceResolver;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+
+@Configuration public class WebMvcConfigure extends WebMvcConfigurerAdapter {
```

Comment:
Again, any time we annotate a class, method, variable, etc, each annotation should be on its own line, followed by class or method etc that it applies to.